shelley/chain-and-ledger/executable-spec/test/Generator/Core.hs

```diff
@@ -0,0 +1,97 @@
+{-# LANGUAGE PatternSynonyms #-}
+
+module Generator.Core
+  ( findPayKeyPair
+  , genTxOut
+  , genUtxo0
+  , traceKeyPairs
+  , someKeyPairs
+  , pickStakeKey
+  , toAddr
+  , toCred
+  , unsafeMkUnitInterval)
+  where
+
+import           Cardano.Crypto.DSIGN (deriveVerKeyDSIGN, genKeyDSIGN)
+import           Crypto.Random (drgNewTest, withDRG)
+import           Data.Maybe (fromMaybe)
+import           Data.Word (Word64)
+import           Hedgehog (Gen)
+import qualified Hedgehog.Gen as Gen
+import qualified Hedgehog.Range as Range
+
+import           BaseTypes (UnitInterval, mkUnitInterval)
+import           Coin (Coin (..))
+import           Keys (pattern KeyPair, pattern SKey, pattern VKey, hashKey, vKey)
+import           LedgerState (genesisCoins)
+import           MockTypes (Addr, Credential, KeyPair, KeyPairs, SKey, TxOut, UTxO, VKey)
+import           Tx (pattern TxOut)
+import           TxData (pattern AddrBase, pattern KeyHashObj)
+
+mkKeyTuple :: (Word64, Word64, Word64, Word64, Word64) -> (VKey, SKey)
+mkKeyTuple seed = fst . withDRG (drgNewTest seed) $ do
+  sk <- genKeyDSIGN
+  return ( VKey $ deriveVerKeyDSIGN sk
```

Comment:
isn't this just `mkKeyPair` from `Examples`?

Reply:
yes, I'm importing from Examples as you suggest (though I think we still need to find a better home than Examples.hs)

Reply:
we now have a `Test.Utils` module which might be a better place (but surely the famous "utils" module isn't the best place)